app/controllers/employer/base_controller.rb

```diff
@@ -1,9 +1,9 @@
 class Employer::BaseController < ApplicationController
```

Comment:
@roscalabrin There can be a benefit to creating a base controller if there is more than one controller that inherits from it and has shared behavior. If it's only for permissions and you have that handled in the services directory it might not be necessary to have the base controller.